test/persistence/compaction/compaction.td

```diff
@@ -35,6 +35,10 @@ true
 # Insert over 1 MiB of data.
 > INSERT INTO compaction VALUES (repeat('this is a compaction test. ', 50000));
 
+# Force advance the timestamp
+> SELECT COUNT(*) FROM compaction;
```

Comment:
You're right, I fixed the bug that also made sqllogictests slow and removed this

Reply:
Hrm I think I was wrong about this because I misunderstood what was happening. This PR doesn't change when tables advance, just how. So this certainly could have been doing something useful.

What was the bug that made it slow? That Append wasn't a batch?

Reply:
I had the code that advanced the timestamp inside the wrong loop so I was advancing it multiple times and in the tests that meant that you'd only get results in the next tick